scripts/exgfs_wave_post_gridded_sbs.sh

```diff
@@ -402,7 +402,7 @@
     cd $DATA
 
     FHINCG=$(( DTFLD_WAV / 3600 ))
-    if [ $fhr = $fhrg ]
+    if [ $fhr = $fhrg ] && ([ "$DOGRB_WAV" = 'YES' ] || [ "$DOGRI_WAV" = 'YES' ]) 
```

Comment:
@NeilBarton-NOAA can you help me understand the updates to this file? I was expecting lines adding  gefsv13_025 but not these.

Reply:
@JessicaMeixner-NOAA DOGRB_WAV and DOGRI_WAV are both no, but the script would still double check for the grib2 file and error out. Specifically errors out on GRDNAME not being defined. This line says only double check for the grib2 files if DOGRB_WAV and DOGRI_WAV is YES.

Reply:
I think the variables are easy enough to follow, but agree on a better solution maybe to remove it from the workflow.  Any thoughts @WalterKolczynski-NOAA ?

Reply:
@NeilBarton-NOAA can you help me understand why we don't see an added line for the new grid near line 327?  I saw it in your earlier scripts but don't see it here.

Reply:
@JessicaMeixner-NOAA I didn't committed it because that part of the script wasn't being used. I'm happy to add. Are the below the correct values?

GRDNAME='global' ; GRDRES=0p50 ; GRIDNR=255  ; MODNR=10

Reply:
Can you send me what atm resolution/IC date you are using so I can run a test on my end? I really thought we needed this line and would like to understand better.

Reply:
> I think the variables are easy enough to follow, but agree on a better solution maybe to remove it from the workflow. Any thoughts @WalterKolczynski-NOAA ?

If at all practical, I'd prefer not to have the job run in the first place if it is not going to do anything.